### ISSUE TITLE
Test suite fails for perl 5.21.x

### ISSUE THREAD

**Issue by eserte** [maxmind/JSON-XS-Sugar-perl#1]
Preliminary matrix view of pass/fail reports: http://217.199.168.174/cgi-bin/cpantestersmatrix.pl?dist=JSON-XS-Sugar%201.00
More information maybe later...

**Comment by eserte** [comment]
Tests run fine until 5.21.4 and start failing with 5.21.5. The log:

```
...
t/02jsontf.t .............. ok
t/03jsonns.t .............. 
Failed 26/26 subtests 
t/author-eol.t ............ skipped: these tests are for testing by the author
...
Test Summary Report
-------------------
t/03jsonns.t            (Wstat: 11 Tests: 0 Failed: 0)
  Non-zero wait status: 11
  Parse errors: Bad plan.  You planned 26 tests but ran 0.
Files=14, Tests=8,  3 wallclock secs ( 0.08 usr  0.09 sys +  1.79 cusr  0.45 csys =  2.41 CPU)
Result: FAIL
Failed 1/14 test programs. 0/8 subtests failed.
```

I see this on freebsd and linux systems.

**Comment by autarch** [comment]
Looks like this was resolved in 1.01
